bigbluebutton-html5/imports/ui/components/tabs/TabsView.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { Fragment } from 'react';
+
+import Tab from './components/Tab';
+import TabContentContainer from './components/TabContent';
+import TabContentVideo from './components/TabContentVideo';
+import TabContentWeb from './components/TabContentWeb';
+import Icon from '/imports/ui/components/Icon';
+import IconButton from '/imports/ui/components/common/IconButton';
+
+const TabsView = ({
+  selectedIndex, tabsCollection, onTabClick, onPresentationClick, onSelectoption, selectedOption,
+}) => {
+  const getComponent = () => {
+    const { fileType } = tabsCollection[selectedIndex];
+
+    switch (selectedIndex) {
+      case 0:
+      case 1:
+        return (
+          <TabContentContainer
```

Comment:
may be Component name would be `TabPresentationContent`

Reply:
component renamed